system-int-test/src/cmr/system_int_test/utils/tag_util.clj

```diff
@@ -231,9 +231,13 @@
    (let [{:keys [status body errors]} response
          expected-tas (map #(coll-tag-association->expected-tag-association % error?)
                            coll-tag-associations)]
-     (is (= [200
-             (set (comparable-tag-associations expected-tas))]
-            [status (set (comparable-tag-associations body))])))))
+     (if (some (fn [element] (contains? element :errors)) (vals coll-tag-associations))
+       (is (= [400
```

Comment:
Let's remove the `if` and duplicate this to make an `assert-tag-association-response-error?` function to handle the 400 case.